core/test/network-records-to-devtools-log.js

```diff
@@ -91,19 +91,36 @@ function getRequestServedFromCacheEvent(networkRecord, index) {
  */
 function getResponseReceivedEvent(networkRecord, index) {
   const headers = headersArrayToHeadersDict(networkRecord.responseHeaders);
-  let timing;
-  if (networkRecord.timing) {
-    timing = {...networkRecord.timing};
-    if (timing.requestTime === undefined) {
-      timing.requestTime = networkRecord.startTime / 1000 || 0;
-    }
-  }
+  // let timing;
+  // if (networkRecord.timing) {
```

Comment:
oops, forgot to remove.

Note that this code has moved. I tried to explain in the comment elsewhere.

Reply:
Kompitabel korleis type i dag